CHANGELOG.md

```diff
@@ -1,7 +1,49 @@
-# 1.0.1
+<!--markdownlint-disable MD012 no-multiple-blanks-->
 
-* Adding Guid to `Carbon.FileSystem.psd1` which is needed to publish to PSGallery.
+# Carbon.FileSystem Changelog
 
-# 1.0.0
+## 1.1.0

```

Comment:
Add release date.

Reply:
No release date yet. This will stay as a prerelease version until then.